action.yml

```diff
@@ -0,0 +1,118 @@
+name: 'Publish to NeoFS Github Action'
```

Comment:
s/`Publish to NeoFS Github Action`/Publish to NeoFS`?

Reply:
> s/`Publish to NeoFS Github Action`/`Publish to NeoFS`?

Fixed